### ISSUE TITLE
Does it only support broadcast mode?

### ISSUE THREAD

**Issue by pokliu** [thomasfla/Linux-ESPNOW#6]
I found that this library works fine when using broadcast mode, but if I configure the target address as `ESP_mac`, then the `esp_now_send` function call returns a non-zero error.

``` c
...
static uint8_t my_mac[6] = {0xA4, 0xC4, 0x94, 0x20, 0x8A, 0x0F};
static uint8_t dest_mac[6] = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
static uint8_t ESP_mac[6] = {0x34, 0x85, 0x18, 0x8F, 0x73, 0x48};
...
handler = new ESPNOW_manager("wlp2s0", DATARATE_1Mbps, CHANNEL_freq_11, my_mac, ESP_mac, false);
handler->set_filter(ESP_mac, my_mac);
...
```

![image](https://github.com/thomasfla/Linux-ESPNOW/assets/7621696/47810e99-a4c2-4e0f-86db-3d2bd3094f92)

From the packet capture results, it seems that Linux-ESPNOW does not have the ACK ESP32 module.

**Comment by thomasfla** [comment]
You are correct, I did not implement the ACK of the espnow protocol and only worked with broadcast packet. 
For our application, we needed low latency, so we used broadcast for this purpose of not having an acknowledgement packet. 

I guess you could implement this using this code as a simple base.
